core/adminserver/admin_server.go

```diff
@@ -45,7 +44,7 @@ var (
 			DisplayName:        "KT SMH Log",
 			TreeState:          tpb.TreeState_ACTIVE,
 			TreeType:           tpb.TreeType_LOG,
-			HashStrategy:       tpb.HashStrategy_OBJECT_RFC6962_SHA256,
+			HashStrategy:       tpb.HashStrategy_RFC6962_SHA256,
```

Comment:
Might be helpful to document that the verifier needs to import whichever hash algorithms you choose to use here.

Reply:
Done